src/components/MemberWall.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Col, Container, Image, Row } from "react-bootstrap";
+
+function MemberWall({ members }) {
+  return (
+    <Container>
+      <h3>Members</h3>
+      <Row>
+        {members.map((member) => (
+          <Col key={member.login} md={4}>
+            <Image src={member.avatar_url} height='100' alt={`${member.login}'s profile picture`}  roundedCircle />
+            <h3>{member.login}</h3>
+          </Col>
+        ))}
+      </Row>
+    </Container>
+  );
+}
+
+export default MemberWall;
```

Comment:
We should probably add some checks to ensure that we add an empty newline at the end of files.

Reply:
#10 resolves this.